src/components/item/item-options.ts

```diff
@@ -39,13 +40,19 @@ export class ItemOptions {
    */
   @Output() ionSwipe: EventEmitter<ItemSliding> = new EventEmitter<ItemSliding>();
 
-  constructor(private _elementRef: ElementRef, private _renderer: Renderer) {}
+  constructor(private _elementRef: ElementRef, private _renderer: Renderer, private _plt: Platform) {}
 
   /**
    * @hidden
    */
   getSides(): number {
-    if (isPresent(this.side) && this.side === 'left') {
+    if (isPresent(this.side) && this.side === 'left' && !_plt.isRTL()) {
```

Comment:
This misses 'right' and `ltr` (I get that you set it as default, just a tiny bit odd. Great work though.

What do you think of?
```
if (isPresent(this.side))
	switch(this.side) {
		case 'left':
			return _plt.isRTL()? ITEM_SIDE_FLAG_RIGHT: ITEM_SIDE_FLAG_LEFT;
		case 'right':
			return _plt.isRTL()? ITEM_SIDE_FLAG_LEFT: ITEM_SIDE_FLAG_RIGHT;
	}

return ITEM_SIDE_FLAG_RIGHT;
```